spec/controller/book_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::BookController, type: :controller do
+  include_context 'User Auth'
+  describe 'GET #index' do
+    context 'When fetching all books' do
+      let!(:book) { create_list(:book, 3) }
+
+      before do
+        get :index
+      end
+
+      it 'response with books json' do
+        resp = JSON.parse(response.body)
```

Comment:
you can use be_paginated of wor-paginate https://github.com/Wolox/wor-paginate#test-helpers